bin/delete_existing_curation.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
+
+# Copyright 2023 EMBL - European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+
+from ebi_eva_common_pyutils.config import cfg
+from ebi_eva_common_pyutils.logger import logging_config as log_cfg
+
+from eva_submission.biosample_submission.biosamples_communicators import WebinHALCommunicator, AAPHALCommunicator
+from eva_submission.biosample_submission.biosamples_submitters import BioSamplesSubmitter
+from eva_submission.submission_config import load_config
+
+logger = log_cfg.get_logger(__name__)
+
+
+class LastSampleCurationDeleter(BioSamplesSubmitter):
+
+    def __init__(self, communicators=None):
+        if not communicators:
+            communicators = LastSampleCurationDeleter.get_config_communicators()
+        super().__init__(communicators, submit_type=('curate',), allow_removal=True)
+
+
+    @staticmethod
+    def get_config_communicators():
+        communicators = []
+        # If the config has the credential for using webin with BioSamples use webin first
```

Comment:
If this is the same as [here](https://github.com/EBIvariation/eva-submission/blob/846cf285e54c634a2a8167086b99f65185895a7e/eva_submission/biosample_submission/biosamples_submitters.py#L238) then it might make sense to refactor so as to not repeat logic.